apps/web/app/api/(internal)/pipeline/lib/handleIntegrations.ts

```
@@ -89,12 +85,7 @@ export const handleIntegrations = async (
         }
         break;
       case "slack":
-        const slackResult = await handleSlackIntegration(
-          integration as TIntegrationSlack,
-          data,
-          survey,
-          contactAttributes
-        );
+        const slackResult = await handleSlackIntegration(integration as TIntegrationSlack, data, survey);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)